portfolio/src/main/webapp/Controller.js

```diff
@@ -0,0 +1,61 @@
+import CommentHandler from '/CommentHandler.js';
+import UserHandler from '/UserHandler.js';
+
+export default class Controller {
+
+  constructor(document) {
+    this.username_ = undefined;
+    this.loggedIn_ = false;
+    this.document_ = document;
+    this.commentHandler_ = new CommentHandler();
+    this.userHandler_ = new UserHandler();
+  }
+
+  async init() {
+    await this.loadComments_(this.document_.getElementById('comments'), 
+                       this.document_.getElementById('comment-cell'));
+    await this.checkLoginStatus_();
+  }
+
+  /** Shows the change name form*/
```

Comment:
Formatting here and below. Please start scanning for these before sending code for review.